WolvenKit/Resources/Scripts/Wolvenkit/Wolvenkit_FileValidation.wscript

```diff
@@ -253,6 +254,9 @@ import * as Logger from 'Wolvenkit/Logger.wscript';
     }
 
     export function validateAppFile(app, validateRecursively) {
+        // invalid app file - not found
+        if (!app) { return; }
```

Comment:
Same as above

Reply:
That's actually legal in JS/TS :D Let's compromise on putting the return on its own line, since inline braces are ok in JS